src/navigation/DrawerNavigator.tsx

```diff
@@ -0,0 +1,45 @@
+import React, {FC} from 'react';
+import {View} from 'react-native';
+import {createDrawerNavigator} from '@react-navigation/drawer';
+import Main from '../screens/Main/Main';
+import CartButton from '../components/TopBar/CartButton';
+import {BLUE, FONT_FAMILY} from '../constants';
+import {DRAWER_ROUTES} from '../constants/routes';
+import CustomDrawerContent from '../components/CustomDrawerContent/CustomDrawerContent';
+
+const Drawer = createDrawerNavigator();
+
+const DrawerNavigator: FC = () => {
+  return (
+    <Drawer.Navigator
+      backBehavior="initialRoute"
+      defaultStatus="closed"
+      screenOptions={{
+        headerStyle: {
+          backgroundColor: BLUE,
+        },
+        headerTintColor: '#fff',
```

Comment:
there are a lot of places with white color. Maybe it will be better to put it in constant?